README.md

```diff
@@ -28,28 +36,30 @@ To install the Biome IntelliJ Plugin, Head over to [official plugin page](https:
 2. Press `⌘Сmd,` to open the IDE settings and then select Plugins.
 3. On the Plugins page, click The Settings button and then click Install Plugin from Disk….
 
-## Biome Resolution
+## Getting Started
+### Biome Resolution
 
 The Plugin tries to use Biome from your project’s local dependencies (`node_modules/.bin/biome`). We recommend adding Biome as a project dependency to ensure that NPM scripts and the extension use the same Biome version.
 
 You can also explicitly specify the `Biome` binary the extension should use by configuring the `Biome CLI Path` in `Settings`->`Language & Frameworks`->`Biome Settings`.
 
-## Plugin settings
+### Plugin settings
 
-### `Biome CLI Path`
+#### `Biome CLI Path`
 
 This setting overrides the Biome binary used by the plugin.
 
-## Supported IDEs
+### <a name="supportedIDEs">Supported IDEs</a>
```

Comment:
```suggestion
### Supported IDEs
```

If the link to the header is `#supported-ides`, resorting to this trick is not necessary.